src/pages/docs/concepts/car.md

```diff
@@ -98,11 +98,11 @@ The [`@ipld/car`](https://github.com/ipld/js-car) package contains the main Java
 Here's a simple example of loading a CAR file from a Node.js stream and storing it with web3.storage:
 
 ```js
-import { createReadStream } from 'fs'
+import fs from 'node:fs'
 import { CarReader } from '@ipld/car'
 
 async function storeCarFile(filename) {
-  const inStream = createReadStream(filename)
+  const inStream = fs.createReadStream(filename)
   const car = await CarReader.fromIterable(inStream)
   const client = makeStorageClient()
   const cid = await client.uploadCAR(car)
```

Comment:
note that this is not correct, but we can update in follow up PR https://github.com/storacha-network/www/issues/94

Reply:
I fix it here ;)